lib/rubygems/commands/pristine_command.rb

```diff
@@ -46,6 +46,12 @@ def initialize
       options[:env_shebang] = value
     end
 
+    add_option('-n', '--bindir DIR',
+               'Directory where binary files are',
```

Comment:
should this be `binstubs` instead of `binary files` ?

Reply:
https://github.com/rubygems/rubygems/blob/672ddcbe14f49c48f6c5492b38480a9162680b5b/lib/rubygems/install_update_options.rb#L27-L31

Reply:
Even if it’s copy-pasted from elsewhere... the “bin” files RubyGems installs aren’t actually binary

Reply:
I'd agree.  The terms 'binary' and 'binstubs' are used both as above and in docs.  What's a better name?  Maybe 'launch scripts', 'CLI scripts', 'scripts', etc.  I don't know...